### PR TITLE
Type fixes from the future.

### PR DESCRIPTION
**Pull request by microbit-matt-hillsdon**
Spotted when experimenting with upgrades but should all be valid now.

### CODE REVIEW

File: src/project/DownloadFlashButton.tsx [hunk 1]
```diff
@@ -86,6 +88,7 @@ const DownloadFlashButton = ({ size }: DownloadFlashButtonProps) => {
                 </MenuItem>
               )}
               <MenuItem
+                as="a"
                 target="_blank"
```
Comment (microbit-robert) [inline src/project/DownloadFlashButton.tsx:92]: These are the download/flash buttons so these shouldn't be links at all. They also lose the button cursor icon with this change. Remove `as`, `target` and `rel` props from all?

**Review by microbit-robert — CHANGES_REQUESTED** [review]

File: src/project/DownloadFlashButton.tsx (hunk 1 above)
Comment (microbit-matt-hillsdon) [reply src/project/DownloadFlashButton.tsx:92]: Oops, yes, thank you.

Comment (microbit-matt-hillsdon) [reply src/project/DownloadFlashButton.tsx]: [caddd1f](https://github.com/microbit-foundation/python-editor-next/pull/564/commits/caddd1f795951080bb08229c1c2b10246d0c5dcb)

File: src/project/DownloadFlashButton.tsx [hunk 2]
```diff
@@ -67,6 +67,7 @@ const DownloadFlashButton = ({ size }: DownloadFlashButtonProps) => {
             <MenuList zIndex={zIndexAboveTerminal}>
               {!connected && (
                 <MenuItem
+                  as="a"
```
[the remark this answers is not in the document]
Comment (microbit-matt-hillsdon) [reply src/project/DownloadFlashButton.tsx]: ca8e892cfd62b3b84ed6e2376cb97beedbb52f2c

File: src/project/DownloadFlashButton.tsx (hunk 1 above)
Comment (microbit-matt-hillsdon) [reply src/project/DownloadFlashButton.tsx]: ... and ca8e892cfd62b3b84ed6e2376cb97beedbb52f2c

**Review by microbit-robert — APPROVED** [review]
LGTM.

**Comment by microbit-matt-hillsdon** [comment]
Merging. @microbit-robert, thanks for spotting the complete mess I'd made of this!

**Pull request merged**